packages/web-client/tests/acceptance/create-card-space-test.ts

```diff
@@ -107,70 +128,61 @@ module('Acceptance | create card space', function (hooks) {
     await click('[data-test-card-space-display-name-save-button]');
     assert.dom('[data-test-card-space-display-name-is-complete]').exists();
 
-    await waitFor(milestoneCompletedSel(1));
-    assert.dom(milestoneCompletedSel(1)).containsText(`Display name picked`);
+    await waitFor(milestoneCompletedSel(2));
+    assert.dom(milestoneCompletedSel(2)).containsText(`Display name picked`);
+
+    // Details
 
-    // // Milestone 3
-    // // Details step
-    await waitFor(postableSel(2, 2));
+    await waitFor(postableSel(3, 2));
     assert
-      .dom(postableSel(2, 1))
+      .dom(postableSel(3, 1))
       .containsText(`Now it’s time to set up your space.`);
     assert
-      .dom(postableSel(2, 2))
+      .dom(postableSel(3, 2))
       .containsText(`Fill out the Card Space details`);
 
     await click('[data-test-card-space-details-start-button]');
     await click('[data-test-card-space-edit-details-save-button]');
 
     assert.dom('[data-test-card-space-details-is-complete]').exists();
 
-    await waitFor(milestoneCompletedSel(2));
+    await waitFor(milestoneCompletedSel(3));
     assert
-      .dom(milestoneCompletedSel(2))
+      .dom(milestoneCompletedSel(3))
       .containsText(`Card Space details saved`);
 
-    // // Milestone 4
-    await waitFor(postableSel(3, 1));
+    // Reservation badge
+    await waitFor(postableSel(4, 1));
     assert
-      .dom(postableSel(3, 0))
+      .dom(postableSel(4, 0))
       .containsText(`We have sent your URL reservation badge`);
-    // // Badge
-    assert.dom(postableSel(3, 1)).containsText(`URL reservation`);
+    assert.dom(postableSel(4, 1)).containsText(`URL reservation`);
```

Comment:
Not related to this PR: Does saying we've sent the reservation badge make sense here? I don't think it does, the user hasn't actually submitted their card space for creation, so we have no reason to send a URL reservation badge, and we don't actually have a URL reservation badge to display... I think this should be taken out of the workflow later until we actually have badges.

Reply:
Yeah good point! I think we can remove it for now. Since I already did some workflow cleanup in this PR I guess I can include the removal here as well.